listener/listener.go

```diff
@@ -29,7 +27,7 @@ type Listener struct {
 	topicName           string
 	subscriptionName    string
 	maxDeliveryCount    int32
-	lockRenewalInterval *time.Duration
+	LockRenewalInterval *time.Duration
```

Comment:
if we can extract the lock renewal from the handler, this can go back to private.

Reply:
done